monai/transforms/lazy/utils.py

```
@@ -192,6 +192,12 @@ def resample(data: torch.Tensor, matrix: NdarrayOrTensor, kwargs: dict | None =
     }
     ndim = len(matrix) - 1
     img = convert_to_tensor(data=data, track_meta=monai.data.get_track_meta())
+    if not (torch.is_floating_point(img) or torch.is_complex(img)):
+        warnings.warn(
```

Ruff (0.16.0)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)